ambari-server/src/main/resources/stacks/BIGTOP/3.2.0/services/HDFS/package/scripts/service_check.py

```diff
@@ -81,12 +81,13 @@ def service_check(self, env):
     params.HdfsResource(None, action="execute")
 
     if params.has_journalnode_hosts:
+      journalnode_web_path = "/journalnode.html"
       if params.security_enabled:
         for host in params.journalnode_hosts:
           if params.https_only:
-            uri = format("https://{host}:{journalnode_port}")
+            uri = format("https://{host}:{journalnode_port}{journalnode_web_path}")
```

Comment:
It'll be better if we can put `/` here between `{journalnode_port}` and `{journalnode_web_path}`

Reply:
Because the variable journalnode_web_path starts with character /,  we don't need  put /  between {journalnode_port} and journalnode_web_path anymore. I will change this iin a different way